cypress/integration/Fields/PDQCitations/index.js

```diff
@@ -0,0 +1,52 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+//This step is just a workaround until it gets fixed
+Given('user is navigating to {string}', (a) => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message like 'list not defined' etc.
+        // and don't want to fail the test so we return false
+        if (err.message.includes(`Cannot read property 'getAttribute' of undefined`)) {
+            return false
+        }
+    });
+    cy.visit(a);
+});
+
+Then('pdq citation section is visible', () => {
+    cy.get('div.pdq-sections > ol').should('be.visible');
+});
+
+And('pdq citation header text is {string}', (headerText) => {
+    cy.get('div.pdq-sections > h6').should('be.visible').first().and('have.text', headerText);
```

Comment:
I don't like how this is only yelding the first citation without making it known in the feature file
if we only checking first then put it in the step
`And pdq citation section number 1 header text is "text"`

Reply:
Done...Changed the step to say "And pdq citation section number 1 header text is "header"